spec/models/post_mover_spec.rb

```diff
@@ -193,7 +205,6 @@
         end
 
         it "moving all posts will close the topic" do
-          topic.expects(:add_moderator_post).twice
```

Comment:
If this is removed, I think we need to add tests to ensure that two moderator posts have been added?